usr/share/rear/lib/global-functions.sh

```diff
@@ -672,10 +672,10 @@ function mount_url() {
             fi
             ;;
         (davfs)
-            mount_cmd="mount $v -t davfs http://$(url_host $url)$(url_path $url) $mountpoint"
+            mount_cmd="mount $v -t davfs http://$(url_host "$url")$(url_path "$url") $mountpoint"
```

Comment:
For consistency, `$mountpoint` should be quoted also here.

Reply:
The `http://$(url_host "$url")$(url_path "$url")` part should probably be quoted.